libraries/SuplaDevice/examples/SuplaSomfy/SuplaSomfy.ino

```diff
@@ -4,6 +4,9 @@
 #include <EEPROM.h>
 #include <supla/io.h>
 
+// WARNING: this example will not work. It requires adjustments to new SuplaDevice library
```

Comment:
If it doesn't work, delete the entire example.

Reply:
I don't want to remove it, becuase it will be last forever ;)

Reply:
done